pyproject.toml

```diff
@@ -67,6 +71,9 @@ oktoberfest = "oktoberfest.__main__:main"
 [tool.poetry.group.dev.dependencies]
 sphinx-autodoc-typehints = ">=1.24,<3.0"
 
+[tool.poetry.extras]
+dlomix = ["dlomix", "tensorflow"]
```

Comment:
check if wandb  is automatically installed with dlomix, if not add it here.

Reply:
It is installed as an optional dependency for DLomix, see https://github.com/wilhelm-lab/oktoberfest/pull/250#discussion_r1755531007